### DISCUSSION TITLE
AttributeError: module 'stumpy' has no attribute 'mdl'

### DISCUSSION THREAD

**Discussion by PatrickKudo** [TDAmeritrade/stumpy#681] [Help: Other Questions] [answered]
Hi, I was trying to follow along the [multidimensional time series profile example](https://stumpy.readthedocs.io/en/latest/Tutorial_Multidimensional_Motif_Discovery.html) and discovered that there doesn't appear to be a mdl (minimum description length) attribute in the stumpy module (1.9.2) I've installed. 

`mdls, subspaces = stumpy.mdl(df, m, motifs_idx, nn_idx)`

Is this a known issue or is there a newer version I should install?

Thanks,
PK

**Comment by seanlaw** [comment, marked as answer]
@PatrickKudo Thank you for your question and welcome to the STUMPY community. `stumpy.mdl` was added on 2022-03-21 in version 1.11.0.  You will need to update your STUMPY version

**Reply by PatrickKudo** [reply]
Great, thank you!
